### PR TITLE
[CC-31] Set up cc-multiverse stage 2

### PR DESCRIPTION
**Pull request by NorrisWu0**
### New Features
- [x] Initialize and set up testing frameworks.
- [x] Integrate `test:coverage` script into husky pre-commit hook
- [x] Configure `webpack` to be able to import SVG components.
- [x] Add mock provider for jest to be able to mock SVG in tests.
- [x] Set up path mapping in `galaxy`, `startrek`, `@cc/ui-chakra`, `@cc/ui-tailwind`

### Bugfix
- [x] Fix reference to `@cc/tsconfig` package in `@cc/ui-chakra` and `@cc/ui-tailwind` package.

### Docs
- [x] Update documentation in regards to running tests.
- [x] Document information on workspace/package name.

### CODE REVIEW

**Comment by NorrisWu0** [comment]
Given the review from @kritomomo, there are changes to be made with the scope of this pr, which is to include the following items.
- Configure `webpack` to be able to import SVG components.
- Add mock provider for jest to be able to mock SVG in tests.
- Document information on workspace/package name.
- Set up path mapping in `galaxy`, `startrek`, `@cc/ui-chakra`, `@cc/ui-tailwind`

File: packages/ui-tailwind/package.json [hunk 1]
```diff
@@ -5,23 +5,27 @@
...
+    "@cc/eslint-config": "*",
+    "@cc/tsconfig": "*",
+    "@types/jest": "^29.1.0",
     "@types/react": "^17.0.37",
     "@types/react-dom": "^17.0.11",
     "autoprefixer": "^10.4.11",
     "babel-loader": "^8.2.5",
     "eslint": "^7.32.0",
-    "eslint-config-custom": "*",
+    "jest": "^29.1.1",
```
Comment (kritomomo) [inline packages/ui-tailwind/package.json:25]: Is this jest dependency really needed?

**Review by kritomomo — APPROVED** [review]
LGTM

File: packages/ui-tailwind/package.json (hunk 1 above)
Comment (NorrisWu0) [reply packages/ui-tailwind/package.json:25]: True, can remove that. 👍

**Review by tatoMa — APPROVED** [review]
Tested everything I know. All good except tw and husky. Nice work.

File: apps/startrek/src/__tests__/pages/Web.test.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import Web from "@src/pages";
```
Comment (tatoMa) [inline apps/startrek/src/__tests__/pages/Web.test.tsx:3]: Name this file Web and Galaxy for the galaxy app?

Comment (NorrisWu0) [reply apps/startrek/src/__tests__/pages/Web.test.tsx:3]: Done done. ✅

**Pull request merged**